src/Components/Blazor/WebAssembly.Authentication/src/Models/AccessToken.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// Represents an access token for a given user and scopes.
+    /// </summary>
+    public class AccessToken
+    {
+        /// <summary>
+        /// Gets or sets the list of granted scopes for the token.
+        /// </summary>
+        public string[] GrantedScopes { get; set; }
```

Comment:
I'm assuming the mutability is because we have to JSON-deserialize this. Is that right?